py/packages/corpora_commander/templates/book.md

```diff
@@ -1,7 +1,8 @@
 {% autoescape off %}
+{% load tex %}
 
 {% for section in project.sections.all %}
-# {{ section.title }}
+# {{ section.title|latex_escape }}
```

Comment:
has to be solved in the image text replace parsing stuff - we have to support $\LaTeX$